src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -357,4 +357,7 @@
   <data name="PackageLoad_InitCodeSnippets" xml:space="preserve">
     <value>Initializing Acumatica Code Snippets shipped with Acuminator</value>
   </data>
-</root>
+  <data name="CodeMap_ExtraInfo_ProjectionDacIndicatorTooltip" xml:space="preserve">
+    <value>This is a projection DAC</value>
```

Comment:
@EAndrosova please review the tooltip for the projection DAC indicator icon.

Reply:
approved